src/components/modules/shared/helpers/ophan.ts

```diff
@@ -0,0 +1,35 @@
+import { OphanComponentEvent, OphanComponentType } from '../../../../types/OphanTypes';
+
+const OPHAN_COMPONENT_ID_OPT_OUT_OPEN = 'article-count-opt-out-open';
+const OPHAN_COMPONENT_ID_OPT_OUT_CLOSE = 'article-count-opt-out-close';
+const OPHAN_COMPONENT_ID_OPT_OUT_CONFIRM = 'article-count-opt-out-confirm';
```

Comment:
Let's also have a `view` event for tracking click rate

Reply:
👍